streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -17,73 +17,72 @@
 
 package org.apache.kafka.streams.kstream;
 
+import org.apache.kafka.common.annotation.InterfaceStability;
 import org.apache.kafka.common.serialization.Serde;
 import org.apache.kafka.streams.KeyValue;
 
 /**
- * KTable is an abstraction of a change log stream from a primary-keyed table.
+ * KTable is an abstraction of a <i>changelog stream</i> from a primary-keyed table.
```

Comment:
For the moment I'd keep the current docstring as is, I think.

That said, I think the description for such an important concept (here: KTable) needs to be more than a 1-liner.  IMHO, basically no user will understand what "KTable is an abstraction of a <i>changelog stream</i> from a primary-keyed table." means unless they have read up on Kafka Streams and the stream-table duality beforehand.

What do you think, @gwenshap ?